wordpress-mobile/WordPress-iOS
# My Site Dashboard (Phase 2): Hook up Quick Action buttons

Part of #17874 

## Description
- Hooked up buttons on the Quick Actions cell on the blog dashboard
   - Stats -> Show stats screen
   - Posts -> Show post list screen
   - Media -> Show media library screen
   - Pages -> Show page screen
- Added event tracking for the Quick Action buttons 

## How to test

Make sure to run your sandbox with the 1.1 API diff applied and to have the My Site Dashboard flag enabled:

1. Go to My Site
2. Go to the dashboard, tap on the **Stats** button
3. ✅ The app should navigate to the Stats screen
4. ✅ The following event should be tracked:`🔵 Tracked: stats_accessed <blog_id: 1, site_type: blog, tap_source: dashboard>`
5. Go back to the dashboard, tap on the **Posts** button
6. ✅ The app should navigate to the Post List screen
7. ✅ The following event should be tracked:`🔵 Tracked: site_menu_opened <blog_id: 1, menu_item: posts, site_type: blog, tap_source: dashboard>`
8. Go back to the dashboard, tap on the **Media** button
9. ✅ The app should navigate to the Media Library screen
10. ✅ The following event should be tracked:`🔵 Tracked: site_menu_opened <blog_id: 1, menu_item: library, site_type: blog, tap_source: dashboard>`
11. Go back to the dashboard, tap on the **Pages** button
12. ✅ The app should navigate to the Page List screen
13. ✅ The following event should be tracked:`🔵 Tracked: site_menu_opened <blog_id: 1, menu_item: pages, site_type: blog, tap_source: dashboard>`

https://user-images.githubusercontent.com/6711616/153590538-deb18cc9-2f13-4e75-8243-31033852c407.mp4


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
